sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -351,6 +362,9 @@ export function createSasTokenProvider(data: {
     sharedAccessSignature: string;
 } | NamedKeyCredential | SASCredential): SasTokenProvider;
 
+// @public
+export const defaultCancellableLock: CancellableAsyncLock;
```

Comment:
I believe it's just a matter of having 1 of these in core-amqp, or 1 in core-amqp, service-bus, and event-hubs. I don't have a strong opinion either way since core-amqp is kind of a weird 'public but not really but it is' package.

Reply:
Arent we replacing the use of `defaultLock` with this `defaultCancellableLock`? If so, I would prefer to maintain the same pattern for now. We can always investigate the caller instantiating in a separate task.